services/types.ts

```diff
@@ -19,11 +33,35 @@ export type Bed = {
 };
 
 export type Plant = {
+  id: string;
+  plantInfoId: string;
   name: string;
   sowDate: string;
   harvestDate: string;
+  logs: PlantLog[];
+};
+
+export type PlantLog = {
+  id: string;
+  creationDate: string;
+  editedDate: string;
+  soilDescription: string;
+  phLevel: number;
   notes: string;
+  photos: string[];
+};
+
+export type PlantInfo = {
```

Comment:
Especially if we start having network requests we won't be getting all of the paint info with each of the users planted plants, it could do with with a better name but I think we'll have to differentiate between the users garden planted plants/logs and the plant info (sun light, water requirements)

Reply:
I see the problem now.
I think we should change the plants tab to 'veggies' or 'vegetables' tab,
change the addPlants component to addVeggies.

I think that will help but we will probably need to look at other naming as you suggested.

Reply:
yeah that's good, it differentiates it a bit. so veggies will be the veggie before its planted and the information about it, and plant will be the users planted plant they'll be adding logs to